### PR TITLE
Clear filters and Change range slider to range input fields #156

### LINKED ISSUE #156: Clear filters on the table views and change number to use min max sliders

### PR DESCRIPTION
**Pull request by joshuadkitenge**
## Description

Add Clear filters button to the catalogue items and manufacturer table

## Testing instructions

Add a set up instructions describing how the reviewer should test the code

- [ ] Review code
- [ ] Check Actions build
- [ ] Review changes to test coverage
- [ ] {more steps here}

## Agile board tracking

closes #156

### CODE REVIEW

File: src/catalogue/items/catalogueItemsTable.component.tsx [hunk 1]
```diff
@@ -375,15 +376,27 @@ const CatalogueItemsTable = (props: CatalogueItemsTableProps) => {
...
-      <Button
-        variant="outlined"
-        onClick={() => {
-          setItemsDialogType('create');
-          table.setCreatingRow(true);
-        }}
-      >
-        Add Catalogue Item
-      </Button>
+      <Box sx={{ display: 'flex' }}>
```
Comment (joelvdavies) [inline src/catalogue/items/catalogueItemsTable.component.tsx:379]: Just wondering if we may want to do it like datagateway, disabling if there are no filters/having icons in them. 
image
I don't mind either way, just thought I would mention it.

File: src/manufacturer/manufacturer.component.test.tsx [hunk 2]
```diff
@@ -103,4 +103,30 @@ describe('Manufacturer', () => {
     const closeButton = screen.getByRole('button', { name: 'Cancel' });
     await user.click(closeButton);
   });
+
+  it('set filters and clears the filters', async () => {
```
Comment (joelvdavies) [inline src/manufacturer/manufacturer.component.test.tsx:107]:
```suggestion
  it('sets the table filters and clears the table filters', async () => {
```
for consistency

**Review by joelvdavies — COMMENTED** [review]
Just a test name could be consistent and a potential suggestion. Also noticed the max doesnt have a button to clear in Cost for catalogue items?
image

File: src/catalogue/items/catalogueItemsTable.component.tsx (hunk 1 above)
Comment (joshuadkitenge) [reply src/catalogue/items/catalogueItemsTable.component.tsx:379]: Should I add the add icon to the add catalogue item and category then ??

**Comment by joshuadkitenge** [comment]
>  image

Got spot the size wasn't big enough

File: src/catalogue/items/catalogueItemsTable.component.tsx (hunk 1 above)
Comment (joelvdavies) [reply src/catalogue/items/catalogueItemsTable.component.tsx:379]: Yes I think so, then its the same as move/copy as well.

**Pull request merged**